modules/seqexec/server/src/main/scala/seqexec/server/ghost/GHOSTHeader.scala

```diff
@@ -0,0 +1,28 @@
+// Copyright (c) 2016-2018 Association of Universities for Research in Astronomy, Inc. (AURA)
+// For license information see LICENSE or https://opensource.org/licenses/BSD-3-Clause
+
+package seqexec.server.ghost
+
+import gem.Observation
+import cats.effect.IO
+import seqexec.model.dhs.ImageFileId
+import seqexec.server.{InstrumentSystem, SeqAction}
+import seqexec.server.keywords._
+import seqexec.server.tcs.TcsKeywordsReader
+
+object GHOSTHeader {
+
+  def header(inst: InstrumentSystem[IO],
```

Comment:
We don't need all the params if the `Header` is empty

Reply:
Can we get rid of all of them?

Reply:
I'm not sure which ones are safe to get rid of at this point... we have TCS keywords, I think, and I'm not sure about ObsKeywords. Do we need a GDSClient?

Reply:
We are not using any of them now. i'm surprsied WartRemover doesn't complaint. i'd remove them all and put them back when needed

Reply:
Sounds good to me!